src/orchestrator-kubernetes/src/lib.rs

```diff
@@ -680,6 +682,51 @@ impl NamespacedOrchestrator for NamespacedKubernetesOrchestrator {
             }]
         });
 
+        let coverage = env::var_os("CI_COVERAGE_ENABLED").is_some();
```

Comment:
We have a policy against env vars in Materialize except at the very edges of the system (command line parsing), as it makes it very difficult to trace the behavior of the system when any component can change its behavior based on env vars, which are basically global mutable state.

Could you instead add a command-line parameter like `--orchestrator-kubernetes-coverage` that is plumbed in into the `KubernetesOrchestratorConfig`? And then here check for whether that config parameter is set to `true`, rather than reading from the environment directly.